apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningService.kt

```diff
@@ -114,20 +113,19 @@ class BeregningService(
                 }
             }
             BehandlingType.MANUELT_OPPHOER -> {
-                val datoFom = foersteVirkFraDoedsdato(grunnlag.hentAvdoed().hentDoedsdato()?.verdi!!)
                 return Beregning(
                     beregningId = UUID.randomUUID(),
                     behandlingId = behandlingId,
                     beregningsperioder = listOf(
                         Beregningsperiode(
                             delytelsesId = "BP",
                             type = Beregningstyper.GP,
-                            datoFOM = datoFom,
+                            datoFOM = virkFOM,
                             datoTOM = null,
                             utbetaltBeloep = 0,
                             soeskenFlokk = listOf(),
-                            grunnbelopMnd = Grunnbeloep.hentGjeldendeG(datoFom).grunnbeløpPerMåned,
-                            grunnbelop = Grunnbeloep.hentGjeldendeG(datoFom).grunnbeløp,
+                            grunnbelopMnd = Grunnbeloep.hentGjeldendeG(virkFOM).grunnbeløpPerMåned,
+                            grunnbelop = Grunnbeloep.hentGjeldendeG(virkFOM).grunnbeløp,
                             trygdetid = 40 // TODO: Må fikses før vi tar imot saker som IKKE har 40 års trygdetid
```

Comment:
Hadde du giddi å tagga denne med mush-have not MVP ? 🤠

Reply:
Går på denne som jeg holder i https://jira.adeo.no/browse/EY-981